excellent/evaluator.go

```diff
@@ -41,9 +41,9 @@ func EvaluateTemplate(env utils.Environment, context types.XValue, template stri
 // EvaluateTemplateValue is equivalent to EvaluateTemplate except in the case where the template contains
 // a single identifier or expression, ie: "@contact" or "@(first(contact.urns))". In these cases we return
 // the typed value from EvaluateExpression instead of stringifying the result.
-func EvaluateTemplateValue(env utils.Environment, context types.XValue, template string, allowedTopLevels []string) (types.XValue, error) {
+func EvaluateTemplateValue(env utils.Environment, context *types.XDict, template string) (types.XValue, error) {
```

Comment:
no such thing as a resolveable anymore - only thing that is resolveable is `XDict`